controlplane/kubeadm/internal/machinefilters/util_test.go

```diff
@@ -0,0 +1,825 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package machinefilters
+
+import (
+	"context"
+	"testing"
+
+	Ω "github.com/onsi/gomega"
```

Comment:
We can avoid this if we only test the public interfaces. To do that, we should change the `package` to something like `machinefilters_test`

Reply:
I think everywhere else we `_ "github.com/onsi/gomega"` import it. Definitely don't want `Ω`, if anything just import it as `gomega`.

Reply:
> We can avoid this if we only test the public interfaces

I want to test private methods as well, this makes a better separation on concern for tests.

> Definitely don't want Ω

Uhm. However, I will fix it